fuse_models/include/fuse_models/unicycle_2d.h

```diff
@@ -123,6 +123,26 @@ class Unicycle2D : public fuse_core::AsyncMotionModel
    */
   bool applyCallback(fuse_core::Transaction& transaction) override;
 
+  /**
+   * @brief Generator function for providing to the \c TimestampManager to create a single motion model segment
```

Comment:
I think this needs to be reworded, maybe something like:
```suggestion
   * @brief Generator function for providing to the \c TimestampManager a way to create a single motion model segment
```